frontend/src/store/modules/tab.ts

```diff
@@ -0,0 +1,105 @@
+import dayjs from "dayjs";
+import { v1 as uuidv1 } from "uuid";
+import { TabInfo, AnyTabInfo, TabState } from "../../types";
+import * as types from "../mutation-types";
+import { makeActions } from "../actions";
+
+export const getDefaultTab = () => {
+  return {
+    id: uuidv1(),
+    label: "Untitled Queries",
+    isSaved: false,
+    savedAt: dayjs().format("YYYY-MM-DD HH:mm:ss"),
+    queryStatement: "",
+    selectedStatement: "",
+  };
+};
+
+// tab store is the sheet store for frontend
+const state: () => TabState = () => {
+  return {
+    tabList: [],
+    activeTabId: "",
```

Comment:
either call both current or active. Please do not mix.

Looks like we use current as in the auth.ts